tests/docker/run_tests.sh

```diff
@@ -24,6 +24,11 @@ die() {
     exit 1
 }
 
+read -p "Rebuild source? (y/n) " rebuild
+if [ "$rebuild" == "y" ]; then
+    ./gradlew clean systemTestLibs
```

Comment:
Not sure it's safe to depend of the local directory. Perhaps it should be relative to `$SCRIPT_DIR`?

I am also not too sure a prompt is the best way. If we have any automated systems using this script, it would fail, right? Could we pass an optional flag instead? Something like `--rebuild` which defaults to false?